packages/mobx/src/core/computedvalue.ts

```diff
@@ -88,7 +88,7 @@ export class ComputedValue<T> implements IObservable, IComputedValue<T>, IDeriva
     lastAccessedBy_ = 0
     lowestObserverState_ = IDerivationState_.UP_TO_DATE_
     unboundDepsCount_ = 0
-    mapid_ = "#" + getNextId()
+    mapid_ = __DEV__ ? "#" + getNextId() : "#"
```

Comment:
Looking at this one, it seems that `mapid_` is used nowhere. Feel free to drop it entirely if that still passes the test

Reply:
Hi, sir. It look likes we can delete it directly?
![2021-02-06 21 46 04](https://user-images.githubusercontent.com/14012511/107119943-a91c9200-68c5-11eb-9dda-7733fd89872c.png)
I search mobx codebase globally and found that nowhere is using this variable ( I am confused 😪

Reply:
If we delete mapid_ directly, The Unit Tests will be failed.
1. keep it 
2. delete it and `jest --updateSnapshot`
Choice which one ? Sir.